src/Language/PureScript/Ide/State.hs

```diff
@@ -194,11 +196,11 @@ populateStage2 :: (Ide m, MonadLogger m) => m ()
 populateStage2 = do
   st <- ideStateVar <$> ask
   duration <- liftIO $ do
-    start <- getCurrentTime
+    start <- getTime Monotonic
     atomically (populateStage2STM st)
```

Comment:
Sorry, ignore me, I read this wrong. 1.5 microseconds is blazingly fast though, are you sure that's right?